### PR TITLE
New standard: doing a timebox

### PR DESCRIPTION
**Pull request by AliceB08**
Début du standard sur les timebox

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/timebox-standard/scrum/timebox.s.md">timebox.s.md</a>**: You failed to mention a *Bad Examples*
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/timebox-standard/scrum/timebox.s.md">timebox.s.md</a>**: You failed to mention a *Good Examples* 
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/timebox-standard/scrum/timebox.s.md">timebox.s.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by yleflour** [comment]
@AliceB08 I feel like you can add more. What is the purpose of a timebox? Why can't we estimate a ticket?

File: scrum/timebox.s.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# [Standard] Doing a timebox
+
+## Owner: Alice Breton
+
+## Checks
+
+### Purpose of a timebox
+It is hard for people to estimate the amount of time they will have to spend on a specific task, but it is much easier to compare the complexity between two tasks. This is why in the scrum method, tickets are estimated according to their complexity: the more difficult they are, the more points they are worth. 
+
+However, to measure the complexity of a ticket the team must know how to develop it. And sometimes they cannot know in advance how to do it because there is uncertainty that needs to be raised. To overcome the impossibilty to estimate the complexity of tickets, we can use a timebox.
```
Comment (Almouro) [inline scrum/timebox.s.md]: there is uncertainty that needs to be raised -> did you mean removed?

File: scrum/timebox.s.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### What is a timebox ?
+In a timebox the team decides to spend a certain amount of time on the ticket.
+At the end of the timebox, there can be several outcomes, here are a few examples:
+- Several solutions were found, it is up to the PR (product owner) to decide which one he would like to see in his project.
+- The developper decided which is the best solution and had the time to partially or completely code the functionality.
+- During the timebox, no solution was found.
+- ...
```
Comment (Almouro) [inline scrum/timebox.s.md]: what is `...`, are there really outher outcomes?
You probably can just remove it

File: scrum/timebox.s.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  - It is interesting to give more detail on the different possibilities that were considered, and why the dev chose this one in particular.
+
+### Good example of a report:
+
+The ticket : (3) [TIMEBOX - 3H] As an admin, when a customer is on the last page of the report and clicks on SEND, I see his appointment updated on the CRM.
+
+
+---------------
+Timebox Report:
+I have raised the uncertainty on sending the request to the CRM
```
Comment (Almouro) [inline scrum/timebox.s.md]: raise?

File: scrum/timebox.s.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+What is left to do:
+
+  - The id of the appointment is stocked in the state of teh `report` page when the user clicks on the button `write a report`
+  - Use this id in the request
+  - Put in the payload of teh request the fields that are in the state of the page `report` to update the report in the CRM
+
+I have added a ticket in the Sprint+1 with a checklist: (2) As a customer, when I click on the `SEND` button my report is added to the CRM
+
+---------------
+
```
Comment (Almouro) [inline scrum/timebox.s.md]: too many lines at the bottom :P

**Pull request merged**